.github/pull_request_template.md

```diff
@@ -0,0 +1,23 @@
+# TITLE
+
+## Description
+
+Description of the pull request changes and motivation.
+
+## Type of change
+
+Please delete options that are not relevant.
+
+- [ ] New feature
+- [ ] Bug fix
+- [ ] Optimization
+
+## Checklist
```

Comment:
Maybe we can add an item here to mark for functionality that could be improved or refactored later on. For instance, rewriting an algorithm to be easier to follow, or to improve its performance. Maybe that can lead to open a new issue which could be linked here as well.